benchmarks/src/main/java/com/ionutbalosin/jvm/performance/benchmarks/macro/signature/DsaEcSignatureBenchmark.java

```diff
@@ -30,7 +30,6 @@
 import java.security.spec.ECGenParameterSpec;
 import java.util.Random;
 import java.util.concurrent.TimeUnit;
-
```

Comment:
I do not know why these signature files were updated ... I suspect the new spotless